CRM/Contribute/BAO/Contribution.php

```diff
@@ -2995,10 +3075,13 @@ public function _assignMessageVariablesToTemplate(&$values, $input, $returnMessa
         $template->assign('customProfile', $customProfile);
       }
 
+      $totalAmount = $this->total_amount;
+
       // for primary contact
       $values['params']['additionalParticipant'] = FALSE;
       $template->assign('isPrimary', 1);
       $template->assign('amount', $primaryAmount);
+      $template->assign('totalAmount', $totalAmount);
```

Comment:
This could be simplified to `$template->assign('totalAmount', $this->total_amount);` ? One less variable.

Reply:
Latest update to PR has addressed this